database_admin/schema/create_schema.sql

```diff
@@ -753,19 +753,15 @@ ON CONFLICT DO NOTHING;
 CREATE TABLE IF NOT EXISTS advisory_metadata
 (
     id               INT GENERATED BY DEFAULT AS IDENTITY,
-    name             TEXT                     NOT NULL,
-    CHECK (NOT empty(name)),
-    description      TEXT                     NOT NULL,
-    CHECK (NOT empty(description)),
-    synopsis         TEXT                     NOT NULL,
-    CHECK (NOT empty(synopsis)),
-    summary          TEXT                     NOT NULL,
-    CHECK (NOT empty(summary)),
-    solution         TEXT                     NOT NULL,
+    name             TEXT                     NOT NULL CHECK (NOT empty(name)),
+    description      TEXT                     NOT NULL CHECK (NOT empty(description)),
+    synopsis         TEXT                     NOT NULL CHECK (NOT empty(synopsis)),
+    summary          TEXT                     NOT NULL CHECK (NOT empty(summary)),
+    solution         TEXT                     NOT NULL CHECK (NOT empty(solution)),
```

Comment:
```
time="2022-06-20T17:19:23Z" level=fatal msg="vmaas data sync failed" err="Failed to sync advisories: Erratas page download and process failed: Storing advisories: Storing advisories: ERROR: new row for relation \"advisory_metadata\" violates check constraint \"advisory_metadata_solution_check\" (SQLSTATE 23514)"
```
either there is a bug in testing data and solution is empty or it is allowed to have empty solution for an erratum

Reply:
`solution` is not present in epel errata - https://mirror.karneval.cz/pub/linux/fedora/epel/8/Everything/x86_64/repodata/e398dc76062bbd82f69ce299da691f254f13c41d4b93df68a6a4effc8dbee385-updateinfo.xml.bz2